client/models.py

```diff
@@ -0,0 +1,26 @@
+from django.db import models
+from django.core.validators import MinValueValidator
+from .constants import LEAD_CATEGORIES, LEAD_CATEGORY_UNCONVERTED
+
+
+
+# Create your models here.
+class Lead(models.Model):
+    # user = models.OneToOneField("accounts.User", on_delete=models.CASCADE, related_name="lead_profile")
```

Comment:
If lead is not user then a 1 to 1 field with customer means that the customer won't be user either. This makes it so neither can ever interact with the system. We should go with making customer a user at least, if not lead.